Makefile

```diff
@@ -1830,6 +1830,10 @@ rustfmt:
 rustfmtcheck:
 	find -name '*.rs' | xargs $(RUSTFMT) --check
 
+# Rust-Analyzer target
+PHONY += rust-analyzer
```

Comment:
Also please add the target to the `Makefile` help output (in the same file, at the bottom).